# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [5]:
s1 = np.random.randint(1,5,100)
s2 = np.random.randint(1,4,100)
s3 = np.random.randint(10000,30000,100)


### Step 3. Let's create a DataFrame by joinning the Series by column

In [16]:
df = pd.DataFrame({'0':s1,'1':s2,'2':s3})

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [17]:
df.columns = ['bedrs','bathrs','price_sqr_meter']

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [31]:
bigcolumn = pd.concat([pd.Series(s1),pd.Series(s2),pd.Series(s3)],axis=0).to_frame()

### Step 6. Oops, it seems it is going only until index 99. Is it true?

0
0        4
1        4
2        1
3        4
4        3
..     ...
295  23714
296  29959
297  21095
298  17508
299  24682

[300 rows x 1 columns]

### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [ ]:
bigcolumn.reset_index(drop=True)